src/components/screens/home/balance-card.tsx

```diff
@@ -0,0 +1,84 @@
+import { getHoursAndMinutes } from "../../../utils/time-utils";
+import { Grid, Typography, Card, CardContent, Skeleton } from "@mui/material";
+import strings from "../../../localization/strings";
+import ScheduleIcon from "@mui/icons-material/Schedule";
+import { errorAtom } from "../../../atoms/error";
+import { useAtomValue, useSetAtom } from "jotai";
+import { useEffect, useState } from "react";
+import { useApi } from "../../../hooks/use-api";
+import { authAtom, userProfileAtom } from "../../../atoms/auth";
+import { PersonTotalTime } from "../../../generated/client";
+
+/**
+ * Component for displaying user's balance
+ */
+const BalanceCard = () => {
+  const userProfile = useAtomValue(userProfileAtom);
+  const { personsApi } = useApi();
+  const setError = useSetAtom(errorAtom);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
verbs should be used only in method

```suggestion
  const [loading, setLoading] = useState(false);
```